src/common/symbols.map

```diff
@@ -453,4 +453,12 @@ MIR_COMMON_2.5 {
     vtable?for?MirCloseSurfaceEvent;
   };
 
-} MIR_COMMON_0.26;
\ No newline at end of file
+} MIR_COMMON_0.26;
+
+MIR_COMMON_2.6 {
```

Comment:
No, current stuff will be in 2.5, we don't need 2.6. (And the rename isn't breaking ABI because we already did that this cycle)